cnd/src/http_api/routes/rfc003/handlers/action.rs

```diff
@@ -53,29 +56,32 @@ pub fn handle_action<D: MetadataStore + StateStore + Network + Spawn + SwapSeed>
                 .select_action(action_kind, method)
                 .and_then({
                     |action| match action {
-                        Action::Accept(action) => serde_json::from_value::<AcceptBody>(body)
+                        Action::Accept(_) => serde_json::from_value::<AcceptBody>(body)
```

Comment:
If we do not use the inner value then should(can?) we remove it from the enum?

Reply:
Yes, that is a possible future refactoring. I'd hesitate to add more complexity to this PR though :)